src/Employer/Employer.Web/Views/ApplicationReview/ApplicationReview.cshtml

```diff
@@ -2,6 +2,15 @@
 @using Esfa.Recruit.Vacancies.Client.Domain.Entities
 @model Esfa.Recruit.Employer.Web.ViewModels.ApplicationReview.ApplicationReviewViewModel
 
+<div asp-condition="@Model.ShowDisabilityStatusAlert" class="grid-row">
+    <div class="column-full">
+        <div class="error-summary small-top-margin" role="alert" aria-labelledby="error-summary-heading-example-2" tabindex="-1" style="border:5px solid #2e358b;">

```

Comment:
There should already be a css class for this border I think.

Reply:
👍